Random seed: 4790955001322091868
DDPG model not found or failed to load. Using default strategy. Error: [Errno 2] No such file or directory: '/Users/james/Desktop/C1GamesStarterKit-master/python-algo/ddpg_actor_model.pth'
---------------- Starting Your Algo --------------------
Configuring your custom DDPG strategy...
Turn 0 using DDPG strategy
DDPG selected action: [0.07451315969228745, 0.05808098986744881, -0.13825391232967377, 0.12431637942790985]
Traceback (most recent call last):
  File "/Users/james/Desktop/C1GamesStarterKit-master/python-algo/algo_strategy_ddpg.py", line 249, in <module>
    algo.start()
  File "/Users/james/Desktop/C1GamesStarterKit-master/python-algo/gamelib/algocore.py", line 74, in start
    self.on_turn(game_state_string)
  File "/Users/james/Desktop/C1GamesStarterKit-master/python-algo/algo_strategy_ddpg.py", line 177, in on_turn
    index_defence = int(round(((defence_vector[1] + 1) / 2) * (len(allowed_positions) - 1)))
IndexError: list index out of range
